numpy/random/mtrand/mtrand.pyx

```diff
@@ -1026,7 +1026,7 @@ cdef class RandomState:
         -----------
         a : 1-D array-like or int
             If an ndarray, a random sample is generated from its elements.
-            If an int, the random sample is generated as if a was np.arange(n)
+            If an int, the random sample is generated as if a was np.arange(a)
```

Comment:
I'd argue that at least `np.arange` should be backticked, but I don't care too much